tests/test_enterprise/api_client/test_enterprise_catalog.py

```diff
@@ -140,3 +140,21 @@ def test_contains_content_items():
     client = enterprise_catalog.EnterpriseCatalogApiClient('staff-user-goes-here')
     actual_response = client.contains_content_items(TEST_ENTERPRISE_CATALOG_UUID, ['demoX'])
     assert actual_response == expected_response['contains_content_items']
+
+
+@responses.activate
+@mock.patch('enterprise.api_client.lms.JwtBuilder', mock.Mock())
+def test_enterprise_contains_content_items():
+    url = _url("enterprise-customer/{enterprise_uuid}/contains_content_items/?course_run_ids=demoX".format(
+        enterprise_uuid=TEST_ENTERPRISE_ID))
```

Comment:
Tiniest possible thing I could complain about, feel free to ignore: give the closing parens their own line.

Reply:
Agreed, makes me wonder if there's a way the linter could catch this